rllib/BUILD

```diff
@@ -1049,6 +1040,15 @@ py_test(
     srcs = ["tests/test_reproducibility.py"]
 )
 
+py_test(
+    name = "test_rollout",
+    main = "tests/test_rollout.py",
+    tags = ["tests_dir", "tests_dir_R", "test_rollout"],
+    size = "enormous",
```

Comment:
Is this test really enormous (that's 1 hour timeout, and it will get retried 3 up to three times)?

We should try to keep any individual test file to less than 5 minutes to avoid travis timeouts.

Reply:
Let me double-check. ...

Reply:
Changed to "large" and removed some unnecessary cont. vs discr. action tests.